tsconfig.json

```diff
@@ -13,7 +13,16 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
-    "noImplicitAny": false
+    "noImplicitAny": false,
+    "baseUrl": "./",
+    "paths": {
+      "@components/*": ["src/components/*"],
```

Comment:
this is a convenience method to avoid all the relative imports from `pages`

Reply:
also moved the code stuff that isn't pages into `src` because the top-level directory was gonna get heinous

Reply:
Just noting that the docs use:

```js
"@/components/*": ["components/*"]
```

I'm 99% sure it's only a preference.

Reply:
I was copying what Dom had done. You could make it “butts” and it would still work so it’s definitely a pref.

Reply:
+1 for "butts"